frontend/src/lib/components/HTMLElementsDisplay/HTMLElementsDisplay.tsx

```diff
@@ -114,16 +127,25 @@ export function HTMLElementsDisplay({
                 </LemonBanner>
             )}
             <div className="px-4 rounded bg-default">
-                {elements.length ? (
+                {parsedElements.length ? (
                     <>
+                        {hasHiddenElements && (
+                            <pre
+                                className="py-2 px-0 m-0 italic text-white text-sm opacity-50 cursor-pointer"
+                                data-attr="elements-display-show-more-of-chain"
+                                onClick={showMoreOfElementsChain}
+                            >
+                                Show hidden elements
```

Comment:
Fly-by: One thing that I think would be very handy here is showing how many levels are hidden, and clarifying that these are hidden _parent_ elements. E.g. "Show 3 more parents" (in that case it would be better if the `hasHiddenElements` bool became a `hiddenParentLevels` number). But that's an extra idea, up to you!

Reply:
I like that! I'll get started on it 🙌🏼

Reply:
Addressed in d64995c68507d0b9e45d43552fdcee3d2ec6a779